routes/__init__.py

```diff
@@ -171,6 +173,9 @@ def logout():
     return redirect(url_for('about'))
 
 
+def domain_from_url(url):
+    urlparse(url).netloc.lower()
```

Comment:
`url` will often not have a protocol (that's the format suggested by the dropdown on the login page), and  `urlparse("example.com")` returns an empty string netloc and a path of "example.com". so we should check for empty netloc and try again with `//` prepended

Reply:
👍 also I should probably have put a `return` there 😅